src/Sulu/Bundle/SecurityBundle/Resources/public/js/services/role-manager.js

```diff
@@ -0,0 +1,186 @@
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define([
+    'services/husky/util',
+    'services/husky/mediator',
+    'sulusecurity/models/role'
+], function(
+    Util,
+    Mediator,
+    Role
+) {
+
+    'use strict';
+
+    var instance = null,
+
+        /**
+         * Saves a role with given data
+         *
+         * @param {Object} data The data to save
+         * @returns {Object} a promise
+         */
+        saveRole = function(data) {
+            var promise = $.Deferred(),
+                role = Role.findOrCreate({id: data.id});
+            role.set(data);
+
+            role.save(null, {
+                success: function(response) {
+                    promise.resolve(response.toJSON());
+                }.bind(this),
+                error: function(context, jqXHR) {
+                    promise.reject(jqXHR);
+                }.bind(this)
+            });
+
+            return promise;
+        },
+
+        /**
+         * Deletes a role for a given id
+         *
+         * @param roleId The id of the role to delete
+         * @returns {Object} a promise
+         */
+        deleteRole = function(roleId) {
+            var role = Role.findOrCreate({id: roleId}),
+                promise = $.Deferred();
+
+            role.destroy({
+                success: function() {
+                    Mediator.emit('sulu.security.role.deleted', roleId);
+                    Mediator.emit('sulu.labels.success.show', 'sulu-security.role.removed');
```

Comment:
Should this service really emit these events? I would say the caller has to know what labels and stuff should be shown.